### ISSUE TITLE
Smoke Detector User's Guide

### ISSUE THREAD

**Issue by gforney** [firemodels/fds#798]
```
Please complete the following lines...

Application Version: 5.3.1
SVN Revision Number: 3182
Compile Date:
Operating System:

Describe details of the issue below:

In the Users Guide released with 5.3.1, I would like to suggest a change to
the smoke detector section, 11.3.4.  As you know, there has been some
confusion about Es vs. Cs in the inputs.  I believe Table 11.1 is correct
per Cleary, IAFSS 6.  You have added a PROP line above as an example, which
is fine.  But in that example, I believe you are trying to put in a Cleary
Ionization I2, but the Es and Cs are swapped.  Therefore, if someone cuts
and pastes out of there, they may not get the Es and Cs correct. 
Therefore, I would change it to say:

&PROP ID='Acme Smoke Detector I2',QUANTITY='spot obscuration',
ALPHA_E=1.8,BETA_E=-1.1,ALPHA_C=1.0,BETA_C=-0.8,ACTIVATION_OBSCURATION=3.28 /

Additionally, there may need to be some more discussion on 'spot
obscuration' vs. 'CHAMBER OBSCURATION'.  For example, at the beginning of
the section, you have QUANTITY='CHAMBER OBSCURATION' for the single
parameter Heskestad model, but later start putting in 'spot obscuration'
for some of the examples.  Perhaps just a small discussion that 'spot
obscuration' is simply 'CHAMBER OBSCURATION' with a default of the SPEC
being soot from the mixture fraction.
```

Original issue reported on code.google.com by `Stephen.Olenick` on 2009-07-22 11:54:26

**Comment by gforney** [comment]
```
Thanks, will do. Actually, 'spot obscuration' and 'CHAMBER OBSCURATION' are the same

thing. We are trying to make these output names more consistent. FDS will still 
accept 'spot', but immediately changes it to the other.
```

Original issue reported on code.google.com by `mcgratta` on 2009-07-22 12:19:02

**Comment by gforney** [comment]
```
Cool.  Then just the easy change to the Es and Cs in the example and maybe just go
with CHAMBER OBSCURATION throughout.  I'll try to remember to check the text you put
in when 5.4.0 is released.  Thanks again.
```

Original issue reported on code.google.com by `Stephen.Olenick` on 2009-07-22 12:30:04

**Comment by gforney** [comment]
```
I added the PROP line you proposed. Turns out we removed 'spot obscuration' a few 
months ago. Thanks -- we'll get this smoke detector stuff right one of these days.
```

Original issue reported on code.google.com by `mcgratta` on 2009-07-24 18:14:04

**Comment by gforney** [comment]
```
Word.  Thanks.  I'll probably find something else I want to change the minute you
release the next manual!
```

Original issue reported on code.google.com by `Stephen.Olenick` on 2009-07-24 19:30:01

**Comment by gforney** [comment]
```
Per your discussion group request, in the FDS User's Guide released with 5.4.2, this
change has been verified.  You can close the issue.  Thanks.
```

Original issue reported on code.google.com by `Stephen.Olenick` on 2009-11-18 21:16:45

**Comment by gforney** [comment]
```
Great -- thanks for checking. K
```

Original issue reported on code.google.com by `mcgratta` on 2009-11-18 21:32:19
